frontend/packages/kubevirt-plugin/src/utils/strings.ts

```diff
@@ -14,12 +14,19 @@ export const SSH = 'ssh';
 export const SYSPREP = 'sysprep';
 export const HARDWARE = 'hardware';
 
-export const RHEL = 'rhel';
-export const CENTOS = 'centos';
+export const RHEL8 = 'rhel8';
+export const RHEL7 = 'rhel7';
+export const CENTOS7 = 'centos7';
+export const CENTOS8 = 'centos8';
+export const WIN2k = 'win2k';
+export const WIN10 = 'win10';
+export const NO_LABEL = 'NO_LABEL';
 
-export const RHEL_EXAMPLE_CONTAINER = 'registry.redhat.io/rhel8/rhel-guest-image';
+export const RHEL8_EXAMPLE_CONTAINER = 'registry.redhat.io/rhel8/rhel-guest-image';
```

Comment:
```suggestion
export const RHEL8_EXAMPLE_CONTAINER = 'registry.redhat.io/rhel8/rhel-guest-image:latest';
```
not specifying a tag is implicitly stating `:latest` so you can either not specifying the tag in all images, or do specify it in all the images, since we did say `:latest` explicitly in all other images, it's nicer to specify it explicitly here too.

Reply:
sure, changed, pushed